streaming/text/convert/enwiki/mds/create_pretraining_data.py

```diff
@@ -19,7 +19,7 @@
 flags.DEFINE_string("input_file", None,
                     "Input raw text file (or comma-separated list of files).")
 
-flags.DEFINE_string("output_dir", None, "Output example directory.")
+flags.DEFINE_string("output_dir", None, "Directory path to store the output MDS shard files")
```

Comment:
let's not diverge from the tfrecord codebase?

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/171/commits/ad49770b6c7f35b3e143cdf0fc6f5f699f7da917.